app/src/androidTest/java/ch/hikemate/app/ui/map/HikeDetailsScreenTest.kt

```diff
@@ -49,6 +52,8 @@ class HikeDetailScreenTest {
           description =
               "A scenic trail with breathtaking views of the Matterhorn and surrounding glaciers.")
 
+  private val detailedRoute = DetailedHikeRoute.create(route)
```

Comment:
Warning: I think doing API calls for testing is a bad practice. You should mock it and hardcode the returned elevations.